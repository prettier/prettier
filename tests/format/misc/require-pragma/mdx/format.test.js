runFormatTest(import.meta, ["mdx"], { requirePragma: true });
