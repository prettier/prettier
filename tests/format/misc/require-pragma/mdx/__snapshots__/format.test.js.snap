// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`with-pragma.md - {"requirePragma":true} format 1`] = `
====================================options=====================================
parsers: ["mdx"]
printWidth: 80
requirePragma: true
                                                                                | printWidth
=====================================input======================================
{/* @prettier */}

I     should      be      formatted       !!

=====================================output=====================================
{/* @prettier */}

I should be formatted !!

================================================================================
`;

exports[`without-pragma.md - {"requirePragma":true} format 1`] = `
====================================options=====================================
parsers: ["mdx"]
printWidth: 80
requirePragma: true
                                                                                | printWidth
=====================================input======================================
I     should      stay      as-is   !!

=====================================output=====================================
I     should      stay      as-is   !!

================================================================================
`;
